.clang-format

```diff
@@ -0,0 +1,72 @@
+---
+Language:        Cpp
+# BasedOnStyle:  LLVM clang-format 4.0
+AccessModifierOffset: -4
+AlignAfterOpenBracket: AlwaysBreak
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlinesLeft: false
+AlignOperands:   true
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: Inline
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterReturnType: None
+AlwaysBreakBeforeMultilineStrings: false
+AlwaysBreakTemplateDeclarations: true
+BinPackArguments: false
+BinPackParameters: false
+BreakBeforeBinaryOperators: None
+BreakBeforeBraces: Stroustrup # Attach
+BreakBeforeTernaryOperators: true
+BreakConstructorInitializersBeforeComma: false
+ColumnLimit:     80
+CommentPragmas:  '^ IWYU pragma:'
+ConstructorInitializerAllOnOneLineOrOnePerLine: false
+ConstructorInitializerIndentWidth: 4
+ContinuationIndentWidth: 8
```

Comment:
there are definitely if conditions that are indented to the same level as the following control flow block in this diff, so AlignOperands either isn't doing that or isn't accomplishing the goal of distinguishing the line continuation from the control flow

if we wanted 8-space indents like the kernel, we'd use that uniformly for all indentation. as it is, it's a mix, mostly 4 spaces, and the parts that use 4 spaces look much better when the next complete line is within the same block.